tests/actions.test.js

```diff
@@ -0,0 +1,22 @@
+import { incrementCount } from '../src/redux/actions';
+import configureStore from 'redux-mock-store';
+
+const mockStore = configureStore();
+const store = mockStore();
+
+describe('test incrementCount action', () => {
+    beforeEach(() => {
+        store.clearActions();
+    });
+    it('dispatches correct action and payload', () => {
+        const expectedActions = [
+            {
+                'type': 'INCREMENT_COUNT',
+                'payload': { updated: (new Date()).valueOf() }
+            },
+        ];
+        store.dispatch(incrementCount({ updated: (new Date()).valueOf() }));
```

Comment:
This provided updated value isn't actually used in the action. There's probably some value in being able to pass in a specific `updated` value, but if it's not used it makes it harder to understand how the action behaves from reading the tests.

Reply:
Would it be better to pass an argument into the action, and set the 'updated' value in the component itself when we dispatch the action? Then in the tests we can test that passing a specific updated value dispatches the action correctly? This is probably better down the line in case users want to log a 'missed' action for example. 

In logButton: where we are setting newDate could pass a predefined date for example
`const mapDispatchToProps = (dispatch) => {
    return {
        incrementCount: () => dispatch(incrementCount((new Date()).valueOf())),
    }
}`

In action:
`export const incrementCount = (updated) => ({
    type: 'INCREMENT_COUNT',
    payload: { updated: updated }
});`

In action.test:
`it('dispatches correct action and payload', () => {
        const expectedActions = [
            {
                'type': 'INCREMENT_COUNT',
                'payload': { updated: 1612235045000 }
            },
        ];
        store.dispatch(incrementCount(new Date('02 Feb 2021 03:04:05 GMT').valueOf()));
        expect(store.getActions()).toEqual(expectedActions);
    });`

Is that what you mean here?

Reply:
Yeah that kind of thing gives us loads of freedom to define when the actions should occur 👍

Reply:
Ok, I've pushed those changes now. Let me know when you have reviewed the style/ test changes and ok to merge.